MekHQ/src/mekhq/campaign/finances/Finances.java

```diff
@@ -239,6 +235,12 @@ public void newDay(Campaign campaign) {
 
         // check for a new year
         if (calendar.get(Calendar.MONTH) == Calendar.JANUARY && calendar.get(Calendar.DAY_OF_MONTH) == 1) {
+            if (campaignOptions.getYearlyFinancesToCSVExport()) {
```

Comment:
I think this would make more sense moving it into the `newFiscalYear` code at the beginning.

Reply:
🤦‍♂ 
Fixed